src/main/java/frc/trigon/robot/subsystems/MotorSubsystem.java

```diff
@@ -110,6 +123,12 @@ public void drive(Measure<Voltage> voltageMeasure) {
     public void updateLog(SysIdRoutineLog log) {
     }
 
+    public void updateMechanisms() {
+    }
+
+    public void updatePeriodically() {
+    }
```

Comment:
Add a very good javadoc to explain everything going on here with `updateMechanism` and the normal `periodic`.